quickstart/201-function-app-node/main.tf

```diff
@@ -0,0 +1,65 @@
+resource "azurerm_resource_group" "default" {
+  name     = "${var.name_prefix}-rg"
+  location = var.location
+}
+
+resource "azurerm_storage_account" "default" {
+  name                     = "${var.name_prefix}sa"
+  resource_group_name      = azurerm_resource_group.default.name
+  location                 = azurerm_resource_group.default.location
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+
+  min_tls_version = "TLS1_2"
+}
+
+resource "azurerm_service_plan" "default_linux" {
+  name                = "${var.name_prefix}-sp"
+  location            = azurerm_resource_group.default.location
+  resource_group_name = azurerm_resource_group.default.name
+  sku_name            = "Y1"
+  os_type             = "Linux"
+}
+
+resource "azurerm_service_plan" "default_windows" {
+  name                = "${var.name_prefix}-sp"
```

Comment:
I've met a name collision due to declaring two azurerm_service_plan with the same name when I try to apply this module, please assign different names for these two plans, thanks.